static/src/stylesheets/base/_type.scss

```diff
@@ -56,7 +56,7 @@ h6 {
 
 
 // SerifText
-.type-5 { @extend %type-5; }
+.type-5 { @extend %type-5 !optional; }
```

Comment:
yeah certainly, but we weren't before, and the older version of node-sass was incorrectly skipping over these bugs, thats' all